clr-k8s-examples/create_stack.sh

```diff
@@ -16,10 +16,13 @@ HIGH_POD_COUNT=${HIGH_POD_COUNT:-""}
 
 # versions
 CANAL_VER="${CLRK8S_CANAL_VER:-v3.9}"
+CILIUM_VER="${CLRK8S_CILIUM_VER:-v1.6}"
 K8S_VER="${CLRK8S_K8S_VER:-}"
 ROOK_VER="${CLRK8S_ROOK_VER:-v1.1.1}"
 METRICS_VER="${CLRK8S_METRICS_VER:-v0.3.5}"
 PROMETHEUS_VER="${CLRK8S_PROMETHEUS_VER:-f458e85e5d7675f7bc253072e1b4c8892b51af0f}"
+CLRK8S_CNI=${CLRK8S_CNI:-"canal"}
```

Comment:
nit: as a pattern we are stripping the CLRK8S for internal vars. `CNI=${CLRK8S_CNI:-"canal"}`